lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -334,16 +333,16 @@ extension JsFlutterConfigurationExtension on JsFlutterConfiguration {
   external JSBoolean? get _canvasKitForceCpuOnly;
   bool? get canvasKitForceCpuOnly => _canvasKitForceCpuOnly?.toDart;
 
-  @JS('canvasKitMaximumSurfaces')
-  external JSNumber? get _canvasKitMaximumSurfaces;
-  double? get canvasKitMaximumSurfaces => _canvasKitMaximumSurfaces?.toDartDouble;
-
   @JS('debugShowSemanticsNodes')
   external JSBoolean? get _debugShowSemanticsNodes;
   bool? get debugShowSemanticsNodes => _debugShowSemanticsNodes?.toDart;
 
   external DomElement? get hostElement;
 
+  @JS('multiViewEnabled')
```

Comment:
We can give any "external" (JS) name we want to this property, like "multiViewEnabledExperimental" or something scary like that. Do we need that, or do we just call it "multiViewEnabled"? @mdebbar?

Reply:
I think it's fine to keep the name `multiViewEnabled` to avoid a little bit of churn down the line. Anyone who makes use of this flag ought to know that this is experimental and could break at any time.

Reply:
I think I agree with this!